capellambse/svg/drawing.py

```diff
@@ -294,35 +288,55 @@ def _draw_label(
             "insert": (builder.label["x"], builder.label["y"]),
             "text_anchor": builder.text_anchor,
             "dominant_baseline": "middle",
-            "style": builder.labelstyle[""],
         }
         if "class" in builder.label:
             textattrs["class_"] = builder.label["class"]
+
+        if transform := getattr(builder.labelstyle, "transform", None):
```

Comment:
Maybe I'm missing a detail in this new branch, but how does this handle multi-line labels? The new code doesn't seem to call any functions related to that. Are those calls not needed here after all?

Reply:
They are probably needed, but I only tested it on single line text labels.

Reply:
We'll simply raise an error for all unsupported cases for now. Something to worry about later. :)